### ISSUE TITLE
Viewer window becomes overcrowded with error messages

### ISSUE THREAD

**Issue by hellocatfood** [NatronGitHub/Natron#459]
### Problem
When an error occurs (usually "contains NaN values") the viewer window can become overcrowded with these messages.

![Screenshot from 2020-02-11 19-02-48](https://user-images.githubusercontent.com/151947/74661971-e8bc0c80-5190-11ea-9811-0142a6d7c2fd.png)

To prevent the viewer window from becoming clogged with error messages can they optionally be rerouted to a separate dock or window?

**Comment by devernay** [comment]
Actually, it would be better to fix what looks like a bug in the retime node.
Those are serious issues. And by the way, can't you use just one retime node at the output of the merge node, instead of all these nodes?
 Can you post a minimalist project (with just one retime) that exhibits this behavior?

**Comment by hellocatfood** [comment]
You're probably right that I could have used one Retime node.

Attached is the same project but with only one source input.

**Comment by devernay** [comment]
I don't have any issue with this project. What version are you running, and on what OS?
(and by the way, this information is part of the issue template for a good reason)

**Comment by hellocatfood** [comment]
You're right, I forgot to include this information. Sorry. I was using 2.3.15 built on 21st January  at the time and have since upgraded to 2.3.15 RC16 where I@m still experiencing this issue.

I use Natron regularly so if I experience this issue again I'll create a separate issue with as much detail as possible.

**Comment by devernay** [comment]
Linux?

**Comment by hellocatfood** [comment]
Yes, Ubuntu 19.10. 64 bit on Dell XPS 15 with GeForce GTX 1050 graphics, 16 gb memory and i7 processor
